### DISCUSSION TITLE
Does Labs work while Media Mod is connected?

### DISCUSSION THREAD

**Discussion by NathanFPerry** [gopro/labs#900] [Q&A]
I have a special use case where I would like to be able to use Labs QR codes to change settings while the Media Mod is connected and working. I have found that QR codes are not read by the camera while the Media Mod is connected. Is that expected behavior? Do the Hero 11/12 work the same? And finally, any chance this can be changed?
Thanks!

**Comment by dnewman-gpsw** [comment]
Labs works with MediaMod attached.  What are you trying to do?

**Reply by NathanFPerry** [reply]
I should have given more detail. When the Media Mod is outputting HDMI to a monitor, then labs QR code read does not work.
If I disconnect the HDMI cable then QR reading works, but with HDMI connected QR reading stops working. 
I have a Hero10 that is always connected to the Media Mod (inside an enclosure that is not easily opened), and I would like to be able to change the settings with Labs QR codes.
Are there any work arounds for this case?

**Comment by NathanFPerry** [comment]
I just found the following comment in the Hero11 notes for labs version: 2.10.70 - Mar 15, 2023

"Added support for QR code scanning will HDMI is connected (the window is narrower, qr code need to be more centered.)"

Is the same true for Hero10? I tried to center it and moved close and far... still no go.

**Reply by dnewman-gpsw** [reply]
That makes sense. As HERO10 hasn't had an update, I doubt that has been fixed if wasn't mentioned in the release notes. While there is a new HERO10 build happening, in QA now, but this wasn't on the list to address. I can't think of a workaround.  Try !MQRDR=1  but I don't have high hopes that this will be sufficient.
